app/models/topic.rb

```diff
@@ -47,6 +41,24 @@ class Topic < ActiveRecord::Base
 
   acts_as_readable
 
+  def self.recent_topics
+    self.find_by_sql %q{
```

Comment:
Redundant `self` detected.<br>Use `%q` only for strings that contain both single quotes and double quotes.